src/diffusers/loaders.py

```diff
@@ -955,6 +955,15 @@ def text_encoder_lora_attn_procs(self):
             return self._text_encoder_lora_attn_procs
         return
 
+    def _remove_text_encoder_monkey_patch(self):
+        for name, _ in self.text_encoder.named_modules():
+            if any(x in name for x in TEXT_ENCODER_TARGET_MODULES):
```

Comment:
Shouldn't this be `TEXT_ENCODER_ATTN_MODULE`? I see this doesn't get reflected in here too:

https://github.com/huggingface/diffusers/blob/523a50a8eb4bf80d2cd622e7d4499c9b69c5f780/src/diffusers/loaders.py#L968

I am asking this because when initializing the LoRA layers for the text encoder, we use `TEXT_ENCODER_ATTN_MODULE`:

https://github.com/huggingface/diffusers/blob/523a50a8eb4bf80d2cd622e7d4499c9b69c5f780/examples/dreambooth/train_dreambooth_lora.py#L864

Doesn't this create a disparity?

Reply:
This code is correct because the target of the monkey-patch is not the `CLIPAttention` itself, but its `q/k/v/out_proj` attributes. However, as you pointed out, I thought it would be easier to understand if it was aligned with `train_dreambooth_lora.py`, so I tried refactoring to remove `TEXT_ENCODER_TARGET_MODULES`. [356a46a](https://github.com/huggingface/diffusers/pull/3649/commits/356a46a6e0932cb7860b19783726ec4aafcfcc6b)  WDYT?